winum.el

```diff
@@ -489,6 +492,7 @@ windows, however a higher number can be reserved by the user-defined
 (defun winum--get-window-vector ()
   "Return the window vector used to get a window given a number.
 This vector is not stored the same way depending on the value of `winum-scope'."
+  (winum--check-for-scope-change)
```

Comment:
The setter is called by `winum--update` which in turn can be called by the minibuffer setup and window config change hooks. A check makes sense here.